internal/contour/translator_test.go

```diff
@@ -462,7 +462,7 @@ func TestTranslatorAddIngress(t *testing.T) {
 			Domains: []string{"*"},
 			Routes: []*v2.Route{{
 				Match:  prefixmatch("/"),
-				Action: clusteraction("default/backend/80"),
+				Action: clusteraction("default/backend/80", map[string]string{}),
```

Comment:
you can write `nil` here and everywhere else in place of `map...`, however I'd prefer it you added a new helper, say `clusteractionrewrite` that took the second map parameter.